lib/ansible/modules/cloud/azure/azure_rm_keyvault.py

```diff
@@ -0,0 +1,430 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2017 Ian Philpot, <ian.philpot@microsoft.com>;
+# Obezimnaka Boms, <t-ozboms@microsoft.com>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: azure_rm_keyvault
+
+version_added: "2.5"
+
+short_description: Create, delete and update key vaults.
+
+description:
+    - Creates, deletes, and updates key vaults.
+
+options:
+    resource_group:
+        description:
+            - name of resource group.
+        required: true
+    name:
+        description:
+            - name of the key vault.
+        required: true
+    state:
+        description:
+            - Assert the state of the key vault. Use 'present' to create or update and
```

Comment:
When referring to values of an option, enclose it in `C()` instead of `'` as the docs will format them correctly. See http://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#formatting-options for more info